src/LocalAuthentication/LAEnums.cs

```diff
@@ -6,19 +6,26 @@ namespace LocalAuthentication {
 
 	[iOS (8,0)]
 	[Mac (10, 10)]
+	[NoTV]
 	[Native]
 	public enum LAPolicy : long {
-		[Mac (10,12,2)]
+		[Mac (10,12,2), NoWatch]
 		DeviceOwnerAuthenticationWithBiometrics = 1,
 		DeviceOwnerAuthentication = 2,
-		[NoiOS][Mac (10,15)]
+		[NoiOS][Mac (10,15)][NoWatch]
 		DeviceOwnerAuthenticationWithWatch = 3,
+		[NoiOS][Mac (10,15)][NoWatch]
+		DeviceOwnerAuthenticationWithBiometricsOrWatch = 4,
+		[Obsolete ("Use DeviceOwnerAuthenticationWithBiometricsOrWatch enum value instead.")]
 		[NoiOS][Mac (10,15)]
-		OwnerAuthenticationWithBiometricsOrWatch = 4,
+		OwnerAuthenticationWithBiometricsOrWatch = DeviceOwnerAuthenticationWithBiometricsOrWatch,
```

Comment:
Shouldn't this one be [NoWatch] as well (because the Obsolete attribute on this value tells you to use another enum value which is obsolete)?